obs-packaging/linux-container/debian.rules

```diff
@@ -18,6 +18,10 @@ override_dh_auto_build:
 	rm -f .config
 	find $(KERNEL_CONFIGS) -name "$(KERNEL_ARCH)_kata_kvm_*" -exec cp {} .config \;
 	[ -f .config ] || (@echo "ERROR: cannot find the kernel config file for the $(KERNEL_ARCH) architecture"; exit 1)
+	# https://github.com/kata-containers/packaging/issues/394 enable reproducible builds:
+	export KBUILD_BUILD_USER=builduser
```

Comment:
Nit: It might be worth making these `katabuilduser` and `katabuildhost`?

Reply:
done